CHANGES.md

```diff
@@ -1,3 +1,24 @@
+## edge-19.6.4
+
+This release adds support for the SMI [Traffic Split](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md)
+API.  Creating a TrafficSplit resource will cause Linkerd to split traffic
+between the specified backend services.  Please see [the spec](https://github.com/deislabs/smi-spec/blob/master/traffic-split.md)
+for more details.
+
+* CLI
+  * Added a check to install to prevent installing multiple control planes into
+    different namespaces
+  * Added support for passing a URL directly to linkerd inject
+  * Added a `--all-namespaces` flag to `linkerd edges`
```

Comment:
The tiniest nit but... I think it should be `an` and not `a`

Reply:
Still tiny, but it seems like we have usually referred to new flags with `the`!